zjit/src/hir.rs

```diff
@@ -707,6 +707,9 @@ pub enum Insn {
     /// Increment a counter in ZJIT stats
     IncrCounter(Counter),
 
+    /// Increment a counter in ZJIT stats for the given unoptimized C function
+    CountUnoptimizedCFunc { signature: String, counter_ptr: *mut u64 },
```

Comment:
Do we care enough about the HIR looking a certain way to make a new opcode that's different from `IncrCounter`? It seems like if not for the printing/asm comment, we could get away with just using that.